cpp/cubes.cpp

```diff
@@ -124,13 +124,13 @@ Hashy gen(int n, int threads = 1)
         return {};
     else if (n == 1)
     {
-        hashes.insert({{XYZ{0, 0, 0}}}, XYZ{0, 0, 0});
+        hashes.insert({{XYZ{0, 0, 0}}});
         printf("%ld elements for %d\n\r", hashes.size(), n);
         return hashes;
     }
     else if (n == 2)
     {
-        hashes.insert({{XYZ{0, 0, 0}, XYZ{0, 0, 1}}}, XYZ{0, 0, 1});
+        hashes.insert({{XYZ{0, 0, 0}}});
```

Comment:
be careful here! this cube is wrong because it is N=1 not N=2

Reply:
woops, my bad, rebasing this onto #5 right now, will fix as I do.